docs/api/QUIC_CONNECTION_EVENT.md

```diff
@@ -278,6 +278,9 @@ This event indicates a certificate has been received from the peer.
 
 Pointer to a platform/TLS specific certificate. Valid only during the callback.
 
+If `QUIC_CREDENTIAL_FLAG_USE_PORTABLE_CERTIFICATES` was specified in the [QUIC_CREDENTIAL_CONFIG](QUIC_CREDENTIAL_CONFIG.md), this will be a `QUIC_BUFFER` containing the DER (binary) encoded remote X.509 certificate.
+
+
```

Comment:
```suggestion
If `QUIC_CREDENTIAL_FLAG_USE_PORTABLE_CERTIFICATES` was specified in the [QUIC_CREDENTIAL_CONFIG](QUIC_CREDENTIAL_CONFIG.md), this will be a `QUIC_BUFFER` containing the DER (binary) encoded remote X.509 certificate.

```
nit: remove extra newline